src/core/response-function.ts

```diff
@@ -13,7 +13,10 @@ export class ResponseFN {
     const content = data;
 
     try {
-      this.data = JSON.parse(this.data).donneesSec;
+      const response = JSON.parse(this.data);
+      if (response.Erreur) throw new Error();
```

Comment:
Can you throw `ServerSideError` instead of `Error`, so that a generic error message will be shown even if all the conditions in the `catch` are falsy ?